libpod/runtime_ctr.go

```diff
@@ -53,6 +60,27 @@ func (r *Runtime) NewContainer(spec *spec.Spec, options ...CtrCreateOption) (c *
 		}
 	}()
 
+	shmdir := ctr.ShmDir()
+	if shmdir == "" {
```

Comment:
Can you set ctr.config.ShmDir here to the new SHM you create? That way we preserve the path in the DB

Reply:
I am setting it about 5 rows below.  ctr.config.ShmDir() return ctr.config.ShmDir,